src/main/resources/dummyResponses/onTrack.json

```diff
@@ -0,0 +1,6 @@
+{
+  "tracking": {
+    "url": "https://track.mock_bpp.com?order_id=order_1",
```

Comment:
th_method is not included

Reply:
it is not included in the schema for ondc it is there in beckn only

Reply:
in the model schema tl_method is not included for ondc